pcsx2/USB/usb-pad/usb-mascon.cpp

```diff
@@ -0,0 +1,585 @@
+// SPDX-FileCopyrightText: 2002-2024 PCSX2 Dev Team
+// SPDX-License-Identifier: GPL-3.0+
+
+#include "usb-mascon.h"
+
+#include "common/Console.h"
+#include "Host.h"
+#include "IconsPromptFont.h"
+#include "Input/InputManager.h"
+#include "StateWrapper.h"
+#include "USB/deviceproxy.h"
+#include "USB/USB.h"
+#include "USB/qemu-usb/USBinternal.h"
+#include "USB/qemu-usb/desc.h"
+
+namespace usb_pad
+{
+	const char* MasconDevice::Name() const
+	{
+		return TRANSLATE_NOOP("USB", "Densha Controller");
+	}
+
+	const char* MasconDevice::TypeName() const
+	{
+		return "DenshaCon";
+	}
+
+	std::span<const char*> MasconDevice::SubTypes() const
+	{
+		static const char* subtypes[] = {
+			TRANSLATE_NOOP("USB", "Type 2"),
+			TRANSLATE_NOOP("USB", "Shinkansen"),
+			TRANSLATE_NOOP("USB", "Ryojouhen"),
```

Comment:
Typo here? In other areas this is read as "Ryojōhen".

Reply:
I wasn't sure whether unicode characters were allowed in source :) Fixed.